build.gradle

```diff
@@ -1,28 +1,23 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
the default project for the last version of Android Studio uses it. I just created a new project and moved to it the usable code, in other words, updated the project to use the last version of the tool.